examples/panel-basic/package.json

```diff
@@ -47,11 +62,11 @@
   },
   "dependencies": {
     "@emotion/css": "^11.1.3",
-    "@grafana/data": "8.3.4",
-    "@grafana/runtime": "8.3.4",
+    "@grafana/data": "latest",
```

Comment:
I think it would be best if we stick to "pinned version numbers" as before rather than the latest keyword. In this way we can see which version of the @grafana packages the plugin is using without needing to look in the yarn.lock file.

Reply:
Good point! 👍 

Fixed in [54fd23b](https://github.com/grafana/grafana-plugin-examples/pull/66/commits/54fd23b0213c5e7357bc59512f6848a794b8d9bb).